sycl/test-e2e/ESIMD/matrix_transpose.cpp

```diff
@@ -5,10 +5,10 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
-// FIXME: Investigate Windows-specific failures
-// REQUIRES: TEMPORARY_DISABLED
-// TODO: esimd_emulator fails due to unimplemented __esimd_oword_ld_unaligned
-// XFAIL: esimd_emulator
+// REQUIRES: gpu
```

Comment:
Please re-base your patch once again.
The current/head version of this test (https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/ESIMD/matrix_transpose.cpp) doesn't need this "REQUIRES: gpu" line
Same for cuda || hip.
Same for gpu-intel-gen9 && windows